RIT-MDRC/Catbot
# Dev environment fix

# Feature
Makes various fixes to allow the program to run on our local (windows) computers.
Notable changes:
Logger uses a different filename when run on dev environment
**!!!** .env file is removed from .gitignore, and added to repo (not sure if this causes problems when pushed to the pi, please review)
Added additional check for is_dev() to detect missing .env file.

# How to Test
Run the cli command "catbot start -D" on your local machine. The GUI should appear. It will crash if you press certain keys, but the main thing this pr is changing is the setup to make sure it works on our computers.
